package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-typescript-basic",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
Please don't modify the package version, there is a release process for this :)

Reply:
fixed, sry :)